src/main/resources/application.yml

```diff
@@ -79,3 +79,5 @@ management:
         enabled: true
       readiness:
         enabled: true
+
+paymentmodality: "01"
```

Comment:
please mention the type of paymentmodality.
"voucherPaymentMoality" or "" defaultPaymentModality

Reply:
paymentmodality changed to defaultPaymentModality